backend/prompt_studio/prompt_studio_v2/models.py

```diff
@@ -0,0 +1,127 @@
+import uuid
+
+from account_v2.models import User
+from django.db import models
+from prompt_studio.prompt_profile_manager_v2.models import ProfileManager
+from prompt_studio.prompt_studio_core_v2.models import CustomTool
+from utils.models.base_model import BaseModel
+
+
+class ToolStudioPrompt(BaseModel):
```

Comment:
@muhammad-ali-e please add the org_id info to this class as well if we decide to add it after [this conversation](https://github.com/Zipstack/unstract/pull/471/files#r1675387307)

Reply:
@chandrasekharan-zipstack This would not be required for now as per mentioned conversation.